metrics/cgroups/oom.go

```diff
@@ -44,11 +44,13 @@ type oomCollector struct {
 }
 
 type oom struct {
+	// count needs to stay the first member of this struct to ensure 64bits
+	// alignment on a 32bits machine (e.g. arm)
```

Comment:
Can you mention this is for `sync/atomic`, even with this explanation it seems odd

Reply:
done